src/components/organisms/OfferPreview.vue

```diff
@@ -0,0 +1,115 @@
+<template>
+  <div class="offer-preview-card">
+    <div class="offer-preview-image">
+      <img :src="offerPreviewImage" alt="Offer image" />
+    </div>
+    <div class="offer-preview-content">
+      <h2 class="offer-preview-title">{{ title }}</h2>
+      <ButtonPrimary class="offer-preview-button" :buttonText="buttonName" :style="{ marginLeft: '0px' }"
```

Comment:
Not a button, also should be conditionally based on the page (can be a boolean prop)